docs/revsets.md

```diff
@@ -88,10 +85,21 @@ revsets (expressions) as arguments.
 * `all()`: All visible commits in the repo.
 * `none()`: No commits. This function is rarely useful; it is provided for
   completeness.
-* `branches()`: All local branch targets. If a branch is in a conflicted state,
-  all its possible targets are included.
-* `remote_branches()`: All remote branch targets across all remotes. If a
-  branch is in a conflicted state, all its possible targets are included.
+* `branches([needle])`: Targets of local branches whose name contains the given
+  string (which is optional and defaults to being empty) -- so, `branches()` is
+  equivalent to `branches("")`. If a branch is in a conflicted state, all its
+  possible targets are included. For example, `branches(push)` would match the
+  branches `push-123` and `repushed` but not the branch `main`.
```

Comment:
nit: Move the sentence with an example before the sentence about conflicts, so it doesn't sound like the example is about conflicts. This also applies to the `remote_branches()` bullet.

Reply:
Done.